tests/labhub_test.py

```diff
@@ -161,14 +162,8 @@ def test_create_issue_cmd(self):
         # Stop ignoring
```

Comment:
@jayvdb ACL's are working as they are supposed to. But it is not actually possible to test ACLs since the message/ACL cannot be mocked from here.
ACL is well tested by the framework as it is an upstream plugin and tested upstream.